frontend/src/metabase/query_builder/components/filters/pickers/TimePicker/TimePicker.jsx

```diff
@@ -36,8 +36,8 @@ const SingleTimePicker = ({ filter, onFilterChange }) => (
 
 SingleTimePicker.horizontalLayout = true;
 
-const MultiTimePicker = ({ filter, onFilterChange, isSidebar }) => (
-  <MultiTimePickerRoot direction={isSidebar ? "column" : "row"}>
+const MultiTimePicker = ({ filter, onFilterChange }) => (
+  <MultiTimePickerRoot direction="row">
```

Comment:
I doubt direction used to do anything if you look at its definition

https://github.com/metabase/metabase/blob/25d1a66109de634ade9c9b33844ee42aff097036/frontend/src/metabase/query_builder/components/filters/pickers/TimePicker/TimePicker.styled.tsx#L13-L26

here's when it was introduced.

https://github.com/metabase/metabase/pull/23398/files#diff-e90e927d0004a5e18592c38b369e3f071618e98327e9f497ec92a39687094f66

Seems like `MultiTimePickerRootProps` are mentioned but never used.

It's also not a valid HTML attribute https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes